homework/index.js

```diff
@@ -29,19 +29,149 @@
     });
     return elem;
   }
+  // ==========start===========
+  /* cSpell: disable */
+  const repoInfo = (index, repoName) => {
+    const container = createAndAppend('div', document.getElementById('root'), {
+      class: 'container',
+      id: 'container',
+    });
+
+    // left div
+    const leftDiv = createAndAppend('div', container, {
```

Comment:
You could name it `repoContainer` instead?

Reply:
OK